src/NuGet.Core/NuGet.Packaging/NuGet.Packaging.csproj

```diff
@@ -9,6 +9,7 @@
     <Shipping>true</Shipping>
     <IncludeInVSIX>true</IncludeInVSIX>
     <XPLATProject>true</XPLATProject>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
Does this have any side effects?  Like does it make the assembly not CLS compliant, will it make the assembly not work with Blazor WASM, something else?

Reply:
I don't believe it changes the compiled IL at all, just gives you a warning by default if you try to use unsafe code.  Enabling is just like signing an agreement that you know what you're doing.